### PR TITLE
add warning about external links

### PR DESCRIPTION
**Pull request by jmartin-tech**
Links to external resources not controlled by the project maintainers are subject to bitrot and malicious take over. Warnings seem appropriate.

## Verification

- [ ] Documentation update only.

Ping to @mdisec, @nicocha30 for added awareness of these changes.

### CODE REVIEW

**Review by smcintyre-r7 — APPROVED** [review]

File: documentation/modules/exploit/linux/http/github_enterprise_secret.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ For testing purposes, you can download a Github Enterprise image from the follow
 
 This module was specifically tested against version 2.8.0, which can be downloaded here:
 
+Download links are provided for reference only and are not maintianed by the project. Utilize at your own risk!
```
Comment (adfoster-r7) [inline documentation/modules/exploit/linux/http/github_enterprise_secret.md:22]:
```suggestion
Download links are provided for reference only and are not maintained by the project. Utilize at your own risk!
```

File: documentation/modules/exploit/linux/http/logsign_exec.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 Download the vulnerable version of OVA or ISO file from following URL. I strongly suggest you to choose OVA.
 
-[http://s3-eu-west-1.amazonaws.com/innotim/Logsign.ova](http://s3-eu-west-1.amazonaws.com/innotim/Logsign.ova)
-[http://s3-eu-west-1.amazonaws.com/innotim/forest-4.4.1-12.04.iso](http://s3-eu-west-1.amazonaws.com/innotim/forest-4.4.1-12.04.iso)
+Download links are provided for reference only and are not maintianed by the project. Utilize at your own risk!
```
Comment (adfoster-r7) [inline documentation/modules/exploit/linux/http/logsign_exec.md:5]:
```suggestion
Download links are provided for reference only and are not maintained by the project. Utilize at your own risk!
```

File: documentation/modules/exploit/linux/http/unraid_auth_bypass_exec.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ performs remote code execution as root by abusing the *extract* function used in
 
 ### Testing Environment
 
+Download links are provided for reference only and are not maintianed by the project. Utilize at your own risk!
```
Comment (adfoster-r7) [inline documentation/modules/exploit/linux/http/unraid_auth_bypass_exec.md:12]:
```suggestion
Download links are provided for reference only and are not maintained by the project. Utilize at your own risk!
```

**Review by wvu — APPROVED** [review]

**Review by wvu — APPROVED** [review]
also hello

**Comment by adfoster-r7** [comment]
## Release Notes

Adds clarification to the module documentation that links to external resources are not controlled by project maintainers. These external resources may no longer exist and are subject to malicious takeover in the future. These links should be reviewed accordingly.

**Pull request merged**